Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -474,7 +469,7 @@ DECLARE_SOA_TABLE_FULL(StoredTracks, "Tracks", "AOD", "TRACK", //! On disk versi
                        track::Sign<track::Signed1Pt>,
                        o2::soa::Marker<1>);
 
-DECLARE_SOA_EXTENDED_TABLE(Tracks, StoredTracks, "TRACK", //! Track parameters at collision vertex
```

Comment:
I assume the extra number can be added already now to the macro (and be ignored?). If yes, I would do it separately, once again to minimize the diff.

Reply:
This can be done.